.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,83 @@
+name: Bug Report
+description: Report a bug encountered while operating Cilium
+labels: ["kind/bug", "needs/triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! If you have usage questions, please try the [slack channel](http://slack.cilium.io/) and see the [FAQ](https://github.com/cilium/cilium/issues?utf8=%E2%9C%93&q=is:issue+label:kind/question+) first.
+
+        **Important**: For security related issues: We strongly encourage you to report security vulnerabilities to our private security mailing list: security@cilium.io - first, before disclosing them in any public forums.
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: |
+        1. In this environment...
+        2. With this config...
+        3. Run '...'
+        4. See error...
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: cilium-version
+    attributes:
+      label: Cilium Version
+      description: What version of our software are you running? (run `cilium version`)
```

Comment:
nit: I think that we should drop the "our" / "your" personal aspects of this, a couple of reasons:
* It can help to remove ego from the equation; it is not "YOUR software caused this problem for me!" it's just "X happened". Remove interpretation or personal bias.
* Cilium is an OSS community. The code is as much our users' as it is the developers' or core team's code. If we encourage users to think of Cilium as also being theirs, then we are one step closer to them considering how to contribute, modify, volunteer to own parts, etc. This project is a shared effort, and we should encode that in the way that we interact with other folks in the community, regardless of how deeply they are involved in the ongoing development.
* Related, rather than giving the impression that folks will file an issue and then it is the responsibility of the maintainers to respond and drive the issue to completion, we should give the impression that this is a place where folks can find, track and coordinate on issues that are affecting them. Then the responsibility to fix it is not purely on the maintainers (although those would be the primary folk to fix the issues), it is a community effort.
```suggestion
      description: What version of the software was running when you discovered this issue? (run `cilium version`)
```